### PR TITLE
放大缩小功能

### PR DESCRIPTION
**Pull request by Dolov**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: examples/example01-includeUi.html [hunk 1]
```diff
@@ -33,7 +33,7 @@
                  },
                  theme: blackTheme, // or whiteTheme
                  initMenu: 'filter',
-                 menuBarPosition: 'bottom'
+                 menuBarPosition: 'left'
```
Comment (junghwan-park) [inline examples/example01-includeUi.html]: I think you should revert position value to `'bottom'`.

File: src/css/main.styl [hunk 2]
```diff
@@ -98,7 +98,7 @@ body > textarea
         position: absolute;
         bottom: 0;
         width: 100%;
-        overflow: auto;
+        overflow: hidden;
```
Comment (junghwan-park) [inline src/css/main.styl]: Why this `overflow` has changed? 
Any reason?

**Comment by junghwan-park** [comment]
@Dolov 
Could you translate this issue to english?
I can't understand your language and your code change means.

**Comment by zhangruinian** [comment]
我觉得 你写的不错, 基础放大缩小可以用, 再加个拖动完美了

**Pull request closed without merging**